models/user.js

```diff
@@ -46,7 +46,7 @@ const userSchema = new Schema(
     },
     photo: String,
     professionalSummary: String,
-    categories: { type: [Schema.Types.ObjectId], ref: CATEGORY },
+    mainSubjects: { type: [Schema.Types.ObjectId], ref: SUBJECT },
```

Comment:
Maybe it would be better to name this field just 'subjects'?

Reply:
I took this name from our UML diagram where we described the user model.